assets/src/components/detours/diversionPage.tsx

```diff
@@ -95,7 +103,7 @@ export const DiversionPage = ({
         <div className="l-diversion-page__panel bg-light">
           {state === DetourState.Edit && (
             <DiversionPanel
-              directions={directions}
+              directions={extendedDirections}
```

Comment:
There is a test that validates the other `directions` -> `extendedDirections` change, but not this one. Would it be possible to add this test as well?